src/components/dialogs/two-windings-transformer/two-windings-transformer-creation-dialog.js

```diff
@@ -438,7 +438,7 @@ const TwoWindingsTransformerCreationDialog = ({
                 isFieldRequired:
                     regulationMode === REGULATION_MODES.CURRENT_LIMITER.id &&
                     phaseTapRegulating,
-                isValueGreaterThan: '0',
+                valueGreaterThan: '0',
```

Comment:
why use string rather than number zero ?

Reply:
String or number, it makes no difference here.
I only changed the name of the param, not its value.